test/acceptance/features/convert-batch-to-transaction.feature

```diff
@@ -0,0 +1,11 @@
+Feature:
+  As a payment service
+  I want to validate and extract the payment requests from batch files and convert them to transactions
+  so that the transactions can be processed in the Payment Service
+
+
+  Scenario: Service will produce a Business Transaction object in JSON Format
```

Comment:
Is `Business Transaction` an appropriate term?  It's not consistent with the language the business use.  Suggest changing to `Payment Request` or `Payment Transaction`.